docs/examples/blocking_consume.rst

```diff
@@ -13,10 +13,10 @@ Example of consuming messages and acknowledging them::
 
 
         def on_message(channel, method_frame, header_frame, body):
-            print method_frame.delivery_tag
-            print body
-            print
-            channel.basic_ack(delivery_tag=method_frame.delivery_tag)
+            print(method_frame.delivery_tag)
+            print(body)
+            print(
+	        channel.basic_ack(delivery_tag=method_frame.delivery_tag))
```

Comment:
Why are you printing the result of basic_ack? Does it return anything interesting? No. That's not what the original code did.